spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -348,7 +334,6 @@ function openUpgradeHandshake(
 
```

Comment:
Typo at line 312: "`openUpgradeHandshake` will open the channel and switch the existing channel parameters to the newly agreed-upon **uprade** channel fields."

Fix: "`openUpgradeHandshake` will open the channel and switch the existing channel parameters to the newly agreed-upon upgrade channel fields."